packages/app-project/src/components/Announcements/components/FinishedAnnouncement/FinishedAnnouncementConnector.js

```diff
@@ -23,7 +23,7 @@ function FinishedAnnouncementConnector() {
 
   const announcement = t('Announcements.FinishedAnnouncement.announcement')
   const link = {
-    href: `${baseUrl}/about/results`,
+    href: `/projects${baseUrl}/about/results`,
```

Comment:
This link should be using the Next router, which automatically adds the app base path.

Reply:
The base path for internal routing, within the Next app, is set here:
https://github.com/zooniverse/front-end-monorepo/blob/4062f399e51d0c63d92856fb03c689e519f32683/packages/app-project/next.config.js#L42

Reply:
#6374 fixes the underlying bug in the `NavLink` component, which was causing this to be rendered as an external link.